daemon/inspect.go

```diff
@@ -58,6 +59,13 @@ func (daemon *Daemon) getInspectData(container *Container) (*types.ContainerJSON
 		FinishedAt: container.State.FinishedAt.Format(time.RFC3339Nano),
 	}
 
+	sizeRwStr, sizeRootFsStr := "N/A", "N/A"
+	if size {
+		sizeRw, sizeRootFs := container.GetSize()
+		sizeRwStr = units.HumanSize(float64(sizeRw))
+		sizeRootFsStr = units.HumanSize(float64(sizeRootFs))
```

Comment:
I'm not sure inspect should return humansize; IIRC, `docker inspect <image>` also returns the raw size. IMO inspect is "low level" data on a container, and I'm not sure "presentation" should be part of that

(But I'm interested to hear if others think different on this)